main_loop.py

```diff
@@ -1,38 +1,65 @@
 import pygame
+import os
+
 from room import Room
 from thing import Thing
 from monster import Monster
 
 
-pygame.init()
+class Game:
+    def __init__(self):
+        self.over = False
+        self.screen = None
+        self.room = None
+        self.time = 0
+        self.font = None
+
+        self.loop()
+
+    def loop(self):
+        pygame.init()
+        self.room = Room(self, 32)
+        self.room.load_from_file(os.path.join("map-files", "level.map"))
+
+        self.screen = pygame.display.set_mode(self.room.canvas_size())
+
+        font_path = os.path.join("fonts", "youmurdererbb_reg.ttf")
+        self.font = pygame.font.Font(font_path, 100)
 
+        while True:
+            for event in pygame.event.get():
+                if event.type == pygame.QUIT:
+                    exit()
 
-r = Room(32)
-r.load_from_file("level.map")
+            for monster in self.room.things_of_class(Monster, static_map=False):
+                monster.step()
 
-screen = pygame.display.set_mode(r.canvas_size())
+            if not self.over:
+                self.room.player.step()
+            elif pygame.key.get_pressed()[pygame.K_SPACE]:
```

Comment:
nechapme preco funguje velmi nespolahlivo skusal som aj KEYUP to slo okus lepsie skusk u teb aci je to iba mojim PC nejaka vada alebo ci je tam nejaka chyba...